templates/repository/common/SECURITY.md

```diff
@@ -21,8 +21,8 @@ https://github.com/ory/meta/blob/master/templates/repository/SECURITY.md
 
 ## Supported Versions
 
-We release patches for security vulnerabilities. Which versions are eligible
-receiving such patches depend on the CVSS v3.0 Rating:
+We release patches for security vulnerabilities. Which versions are eligible for
+receiving such patches, depends on the CVSS v3.0 Rating:
```

Comment:
Are you sure about adding this comma? It reads better to me without...

```suggestion
We release patches for security vulnerabilities. Which versions are eligible for
receiving such patches depends on the CVSS v3.0 Rating:
```

Reply:
Yeah, I think grammarly got confused with the missing "s" and suggested the comma. After I added the "s" I didn't go back to the comma.